registry.k8s.io/hack/initial-copy-between-s3.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REGIONS=(
+    ap-northeast-1
+    ap-south-1
+    ap-southeast-1
+
+    eu-central-1
+    eu-west-1
+
+    us-east-1
+    us-east-2
+    us-west-1
+    us-west-2
+)
+
+SOURCE=s3:prod-registry-k8s-io-us-east-2
+
+CALLER_ID="$(aws sts get-caller-identity --output json | jq -r .UserId)"
+
+for REGION in "${REGIONS[@]}"; do
+    DESTINATION="s3dest:prod-registry-k8s-io-${REGION:-}"
```

Comment:
here and throughout...

little nit: you know that `$REGION` will be defined already so no reason to use the defaulting syntax of `${REGION:-}`. You can just use `$REGION`

Reply:
I write it like `${REGION:-}` instead of `$REGION` since sometimes `shellcheck` likes to have a fallback value, which is allowed to be empty. Something around variable declaration maybe?